jsonnet/node-exporter.jsonnet

```diff
@@ -104,6 +104,9 @@ local tlsVolumeName = 'node-exporter-tls';
               volumes+: [volume.fromSecret(tlsVolumeName, 'node-exporter-tls')],
               securityContext: {},
               priorityClassName: 'system-cluster-critical',
+              tolerations: [
+                { operator: "Exists" },
```

Comment:
this should be even in kube-prometheus no?

Reply:
That makes sense. Here's a PR: https://github.com/coreos/kube-prometheus/pull/126